pkg/gopass/secret/mime.go

```diff
@@ -80,7 +83,10 @@ func ParseMIME(buf []byte) (*MIME, error) {
 	tpr := textproto.NewReader(r)
 	m.Header, err = tpr.ReadMIMEHeader()
 	if err != nil {
-		return nil, &PermanentError{Err: err}
+		if !errors.Is(err, io.EOF) {
```

Comment:
Shoot I should have catched it in #1579 

I actually thought I did: https://github.com/gopasspw/gopass/blob/10e2e0ece4717a1be986ed8cb076d7da08d46486/pkg/gopass/secret/mime.go#L82-L83

I don't understand where this disappeared.